### PR TITLE
[torch.compile] Adding torch compile annotations to vision-language models

### PR DESCRIPTION
**Pull request by CRZbulabula**
This PR is to improve [#9866](https://github.com/vllm-project/vllm/pull/9866), adding torch compile annotations to some vision-language models and testing them.

### CODE REVIEW

File: vllm/model_executor/models/registry.py [hunk 1]
```diff
@@ -138,13 +134,17 @@ def add_embedding_models(base_models, embedding_models):
...
     "Qwen2VLForConditionalGeneration": ("qwen2_vl", "Qwen2VLForConditionalGeneration"),  # noqa: E501
     "Qwen2AudioForConditionalGeneration": ("qwen2_audio", "Qwen2AudioForConditionalGeneration"),  # noqa: E501
     "UltravoxModel": ("ultravox", "UltravoxModel"),
     # [Encoder-decoder]
+    "Blip2ForConditionalGeneration": ("blip2", "Blip2ForConditionalGeneration"),
+    "ChameleonForConditionalGeneration": ("chameleon", "ChameleonForConditionalGeneration"),  # noqa: E501
+    "ChatGLMModel": ("chatglm", "ChatGLMForCausalLM"),
+    "ChatGLMForConditionalGeneration": ("chatglm", "ChatGLMForCausalLM"),
     "MllamaForConditionalGeneration": ("mllama", "MllamaForConditionalGeneration"),  # noqa: E501
+    "PixtralForConditionalGeneration": ("pixtral", "PixtralForConditionalGeneration"), # noqa: E501
```
Comment (DarkLight1337) [inline vllm/model_executor/models/registry.py:147]: By encoder-decoder here, we mean that the model uses the encoder-decoder model runner, so only Mllama currently fits in that category.

Comment (CRZbulabula) [reply vllm/model_executor/models/registry.py]: Sorry for my misunderstanding and thaks for the heads up. The incorrect modifications have been reverted.

**Review by DarkLight1337 — APPROVED** [review]

**Review by youkaichao — COMMENTED** [review]
thanks for the contribution! will review and test tomorrow.

**Comment by DarkLight1337** [comment]
Model changes look fine, I'll leave it to @youkaichao to test them.

**Review by youkaichao — APPROVED** [review]

**Comment by youkaichao** [comment]
thanks for the great efforts!

**Pull request merged**